src/App.js

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'react';
+import { useSelector, useDispatch} from 'react-redux'
+import { MProvider} from './store/redux_store'
+import {fetchPokemons} from './store/slicePokemons'
+import { Navigator} from './components'
```

Comment:
Try formatting the code, you need some extra spaces in all this: 
```suggestion
import { useSelector, useDispatch } from 'react-redux'
import { MProvider } from './store/redux_store'
import {fetchPokemons } from './store/slicePokemons'
import { Navigator } from './components'
```

Reply:
I suggest you to check how to use 'Prettier' in your project, or you can just use the VSCode formatter